tf2/CMakeLists.txt

```diff
@@ -1,54 +1,53 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(tf2)
 
-find_package(console_bridge REQUIRED)
-find_package(catkin REQUIRED COMPONENTS geometry_msgs rostime tf2_msgs)
-find_package(Boost REQUIRED COMPONENTS signals system thread)
+if(NOT WIN32)
+  add_definitions(-std=c++11)
+endif()
 
-catkin_package(
-   INCLUDE_DIRS include
-   LIBRARIES tf2 
-   DEPENDS console_bridge
-   CATKIN_DEPENDS geometry_msgs tf2_msgs rostime)
+find_package(ament_cmake)
+find_package(console_bridge REQUIRED)
+find_package(geometry_msgs REQUIRED)
 
-include_directories(include ${catkin_INCLUDE_DIRS} ${console_bridge_INCLUDE_DIRS})
-include_directories (src/bt) 
+include_directories(include src/bt ${geometry_msgs_INCLUDE_DIRS} ${console_bridge_INCLUDE_DIRS})
 
 # export user definitions
 
 #CPP Libraries
 add_library(tf2 src/cache.cpp src/buffer_core.cpp src/static_cache.cpp)
-target_link_libraries(tf2 ${Boost_LIBRARIES} ${catkin_LIBRARIES} ${console_bridge_LIBRARIES})
-add_dependencies(tf2 tf2_msgs_gencpp)
+target_link_libraries(tf2 ${geometry_msgs_LIBRARIES} ${console_bridge_LIBRARIES})
 
 install(TARGETS tf2
-  ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+  ARCHIVE DESTINATION lib
+  LIBRARY DESTINATION lib
+  RUNTIME DESTINATION lib/${PROJECT_NAME}
```

Comment:
I agree with @wjwwood . `CATKIN_PACKAGE_BIN_DESTINATION` was wrong in the first place and wouldn't have worked on Windows.